src/traversals/recdfs.jl

```diff
@@ -0,0 +1,30 @@
+
+
+# Function to compute all simple paths between source-sink pairs
+
+# Usage: pathSet = []
+#        visited = [source]
+#        RecDFS(graph, visited, sink, pathSet)
+
+function recdfs(graph, visited, sink, pathSet)
+  nodes = LightGraphs.neighbors(graph, visited[length(visited)])
+
+  for i in nodes
+
+    if in(i, visited)
+      continue
+    end
+
+    if i == sink
+      push!(visited, i)
+      push!(pathSet, visited)
+      pop!(visited)
+    elseif !in(i, visited)
+      push!(visited, i)
+      recdfs(graph, visited, sink, pathSet) # Recursion starts here
+      pop!(visited)
+    end
+
+  end
```

Comment:
Consider creating a "starter" function that takes the graph and a source vertex as its only parameters and then calls the recursive function with the appropriate initialized arguments.